.github/workflows/ci.yml

```diff
@@ -90,6 +90,11 @@ jobs:
               body: "Visual regression tests failed 😢. You can download the failure diffs from the 'Artifacts' section of the failed CI run. To update the tests, run `npm run jest:visual-tests:update` locally."
             })
 
+      - name: Percy Test
+        run: npx percy storybook ./dist-storybook
```

Comment:
Worth noting here by using npx it will install the latest from npm rather than using the version specified locally, so could in the future cause any issues if say percy/cli released a major that we need to change.

One thing I would recommend is like when testing locally adding an npm command in the package file to run the installed version :)